### ISSUE TITLE
WARNING: PIE does not support background-image.

### ISSUE THREAD

**Issue by ccorcos** [Igosuki/compass-mixins#68]
No idea what this is all about...

```
WARNING: PIE does not support background-image. Use @include background(linear-gradient(top, #ffffff 0%, #f5f5f5 100%)) instead.
	on line 74 of node_modules/compass-mixins/lib/compass/css3/_images.scss
	from line 84 of affirm-ng/src/styles/auth.scss
```

**Comment by sir-dunxalot** [comment]
I am also seeing this issue with `0.12.7` after installing `compass-mixins` using NPM. Interesting, the warning wasn't present when I installed it via Bower.

**Comment by ijy** [comment]
I don't suppose anyone's found a resolution to this yet have they?

**Comment by ccorcos** [comment]
I'm using bourbon now instead ;)

**Comment by ijy** [comment]
Well that's one solution at least. :) 

The problem seems to stem to the Compass core library in how the background mixin is applied(?)

**Comment by indrekots** [comment]
I'm guessing this is still an ongoing issue?

**Comment by haluvibe** [comment]
Yes, I'm also experiencing this issue. Would be great to get a resolution :)

**Comment by ccorcos** [comment]
Solution is -- dont use compass!

**Comment by dankelleher** [comment]
This is still an issue - I don't think it should be closed.

**Comment by gfdesign** [comment]
I have the same issue.
Has been any solution for this?

**Comment by xzyfer** [comment]
This project isn't actively maintained. We'll accept PRs to fix this however I suggest looking at a modern alternative like bourbon.io or autoprefixer.
